### PR TITLE
Move Installation [started|finished] close to the progress bar

### PR DESCRIPTION
**Pull request by mareklibra**

### CODE REVIEW

File: src/components/clusterDetail/ClusterProgress.tsx [hunk 1]
```diff
@@ -52,13 +65,23 @@ const ClusterProgress: React.FC<ClusterProgressProps> = ({ cluster }) => {
...
+        <DetailItem
+          title="Installation started on"
+          value={getHumanizedDateTime(cluster.installStartedAt)}
+        />
+        <DetailItem title="Installation status" value={getInstallationStatus(cluster)} />
+      </DetailList>
+      <Progress
+        value={progressPercent}
+        label={label}
+        title=" "
```
Comment (mareklibra) [inline src/components/clusterDetail/ClusterProgress.tsx:79]: to avoid inlining of the icon ...

**Comment by mareklibra** [comment]
The details section is unbalanced now, will fix it.

**Comment by andybraren** [comment]
I think this looks good! Having these start/end times near the top certainly seems like a more natural place to look than in the Details section down below. WDYT @nirfarkas? [MGMT-1555](https://issues.redhat.com/browse/MGMT-1555) for reference.

**Comment by nirfarkas** [comment]
It looks much better! One tiny remark: I suggest removing the word 'Installation' from the sub-headers 'Installation stated on' & 'Installation status' because its obvious we're showing info regarding the installation.

**Comment by mareklibra** [comment]
@nirfarkas , done

**Comment by mareklibra** [comment]
After recent changes:

**Comment by nirfarkas** [comment]
Thanks @mareklibra !

**Pull request merged**